keanu-project/src/test/java/io/improbable/keanu/e2e/regression/DiabetesLinearRegression.java

```diff
@@ -57,6 +57,10 @@ public void doesLinearRegressionOnBMIAsModel() {
             .withRegularization(RegressionRegularization.RIDGE)
             .withPriorOnWeightsAndIntercept(0, 100)
             .build();
+
+        linearRegressionModel.observe();
```

Comment:
Since we do `observe()` then `fit()` so much, I wonder whether it would be worth creating an `observeAndFit()`.

Reply:
Reduce this to just `fit()` - `observe` happens as part of the build